docs/1.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE=HTML>
+<link rel="stylesheet" type="text/css" href="style.css">
+
+<center>
+<a href="/">←Back</a>
+
+<h1 id="steps">Steps</h1>
+
+Click <a href="https://github.com/DamianSwanAAJHS2/SillyLittleFiles/releases/download/Pipeline/silly.onc">this link</a> (You should have just gotten a file downloaded)<br>
+Go to <code>chrome://network</code><br>
+This funny thing only works on chromebooks running chrome<br>
+
+<br>
+Scroll to the bottom and press <code>Import ONC</code>, then upload the onc file<br>
+Delete the file and then go to your quick access settings (click the clock) <br>
+Click <code>VPN</code>, and then <code>Silly little file</code> (If unmodified from this original upload)<br>
+Please note this may take a few tries to connect, but be persistant, it will work eventually<br>
+<br>
+Profit<br>
+
+<p>Check out <a href=/11.html> if you are interested in making your own files 😜 </p>
```

Comment:
Missing ending </a> tag, this causes the Sync counter to also become a link element